test/functional/test_framework/test_node.py

```diff
@@ -61,9 +61,13 @@ def __getattr__(self, *args, **kwargs):
         assert self.rpc_connected and self.rpc is not None, "Error: no RPC connection"
         return self.rpc.__getattr__(*args, **kwargs)
 
-    def start(self):
+    def start(self, extra_args, stderr=None):
```

Comment:
`TestNode.start` is only ever called by `TestFramework.start_node`, which has its own default for `extra_args`, so not strictly necessary, but I agree having a default here makes it clearer. Added.